### PR TITLE
Update some dependencies, remove heapsize from the build

### PR DESCRIPTION
**Pull request by SimonSapin**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/19510)

### CODE REVIEW

**Comment by SimonSapin** [comment]
This uses a webrender commit that has not been merged yet: https://github.com/servo/webrender/pull/2192

File: Cargo.lock [hunk 1]
```diff
@@ -3062,11 +3051,6 @@ name = "smallbitvec"
 version = "1.0.6"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 
-[[package]]
```
Comment (Eijebong) [inline Cargo.lock]: This is deduplicating smallvec. It should be removed from the `[ignore]packages` list of servo-tidy

Comment (SimonSapin) [reply Cargo.lock]: Oops, thanks. Fixed that and import order to make tidy happy.

**Pull request merged**